vertx-pg-client/src/main/java/io/vertx/pgclient/PgException.java

```diff
@@ -66,7 +67,7 @@ private static String formatMessage(String errorMessage, String severity, String
   private final String constraint;
 
   public PgException(String errorMessage, String severity, String code, String detail) {
-    super(formatMessage(errorMessage, severity, code));
+    super(formatMessage(errorMessage, severity, code), 0, code);
     this.errorMessage = errorMessage;
```

Comment:
- `getCode()` should be deprecated in favour of `getErrorCode`()
- the `code` field should be removed and `getCode()` should return `getErrorCode()`

Reply:
Done but, in fact, in Pg client `code` is the equivalent of `sqlState`, not `errorCode`.